### PR TITLE
Ei 1709 ytd validation figures

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql [hunk 1]
```diff
@@ -33,6 +33,32 @@ SELECT PaymentYear as [(Paste into cell A6) PaymentYear],PaymentPeriod, count(*)
...
+	FROM incentives.Payment
+	WHERE PaymentYear = @AcademicYear
+	AND PaidDate IS NOT NULL
+	UNION ALL
+	SELECT SUM(Amount)
+	FROM incentives.PendingPayment
+	WHERE Id 
+	IN
+	(
+	SELECT DISTINCT PendingPaymentId FROM incentives.PendingPaymentValidationResult
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql:56]: Shouldn't this be only for failed validation results?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql:56]: The query takes the value of all the pending payments for the academic year that have been validated, and then subtracts the value of all the ones that have passed validation and been paid. The remainder are the ones that have been validated but not paid, so no filter required, I think

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql:56]: OK.  Can you add a comment to the source code s it's clear in the future?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql:51]: Done

**Pull request merged**